config-windows.md

```diff
@@ -19,6 +19,34 @@ The Windows container specification uses APIs provided by the Windows Host Compu
     }
 ```
 
+## <a name="configWindowsDevices" />Devices
+
+**`devices`** (array of objects, OPTIONAL) lists devices that MUST be available in the container.
+
+Each entry has the following structure:
+
+* **`id`** *(string, REQUIRED)* - specifies the device which the runtime MUST make available in the container.
+* **`id_type`** *(string, REQUIRED)* - tells the runtime how to interpret `id`. Today, Windows only supports a value of `class`, which identifies `id` as a [device interface class GUID][interfaceGUID].
```

Comment:
Are there any other types than `class`? 

Also this should be `idType` not `id_type`

Reply:
I also wonder if this is really flexible enough for future needs -- will all future resources that might count as devices have some kind of ID? (I guess it's Windows we're talking about, so everything has a GUID of some kind?)

Being able to attach all sound devices to a container is neat, but if you want something like raw hard drive or USB drive access you'd have to attach _all_ drives, which isn't great, right? (I'm guessing that's why `idType` is included here -- so `id` could instead point to the GUID of an individual device at some point in the future?)

Reply:
@crosbymichael Thanks for catching the typo, let me commit a fix. In the future, we'll certainly have more types. 

That's correct, @tianon. Each device in Windows is guaranteed to have a unique device instance path ID. In the future it would be easy to extend so that id=<some dev instance path ID> and idType="instance" or some other label. Given this, I think what we have is extensible to give finer granularity in defining a device for a Windows container in the future.